thunder/common.py

```diff
@@ -53,6 +54,40 @@
 # TODO RC1 Update last_executed to last_computation
 # TODO RC1 Review how autograd traces are presented
 class CompileStats:
+    """A class holding statistics and caches for a compiled function.
+
+    Attributes:
+        last_executed:
```

Comment:
Thank you for looking at these. Could you add a comment that it is preferable to use the top-level thunder.last_traces etc. functions where applicable, please?